docs/sources/administration/data-source-management/teamlbac/_index.md

```diff
@@ -26,8 +26,8 @@ LBAC for data sources is currently generally available for `Loki` and in **exper
 
 | Data source | [Grafana Cloud](/docs/grafana-cloud) | [Grafana Enterprise](../../../introduction/grafana-enterprise/) |
 | ----------- | ------------------------------------ | --------------------------------------------------------------- |
-| Loki        | GA                                   | GA                                                              |
-| Prometheus  | PrivatePreview                       | PrivatePreview                                                  |
+| Loki        | GA                                   | GA + GEL                                                        |
```

Comment:
The line above should be changed to 
> LBAC for data sources is currently generally available for `Loki` and in **public preview** for `Prometheus`.

I also find the addition of `+ GEL` and `+ GEM` pretty confusing. Maybe putting GEM/GEL under data source column makes a bit more sense?

```suggestion
| Loki and GEL        | GA                                   | GA                                                        |
```

Reply:
I have emphasized the need for enterprise to have GEL/GEM

```md

| Data source | Grafana Cloud | Grafana Enterprise           |
| ----------- | ------------- | ---------------------------- |
| Loki        | GA            | GA (requires GEL)            |
| Prometheus  | PublicPreview | PublicPreview (requires GEM) |
```